### PR TITLE
API v2: documentation for `applyTemplate`

### PR DESCRIPTION
**Pull request by jodosha**
@weppos @jacegu Please review. Thanks! 👍

### CODE REVIEW

File: content/v2/domains/templates.markdown [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+    POST /:account/domains/:domain/templates/:template
+
+### Parameters
+
+Name | Type | Description
+-----|------|------------
+`:account` | `integer` | The account id
+`:domain` | `string`, `integer` | The domain name or id
+`:template` | `integer` | The template id
```
Comment (weppos) [inline content/v2/domains/templates.markdown]: As far as I remember, the template ID can be either the ID or short name. Am I wrong?

Comment (jodosha) [inline content/v2/domains/templates.markdown]: Right: 72398eb

File: content/v2/domains/templates.markdown [hunk 2]
```diff
@@ -0,0 +1,37 @@
+---
+title: Templates API | Domains | DNSimple API v2 (Beta)
+excerpt: This page documents the DNSimple templates API v2.
```
Comment (jacegu) [inline content/v2/domains/templates.markdown]: Being picky I'd say something like: _DNSimple API v2 template documentation_ (avoiding the _This page_ part which makes me uneasy).

Comment (weppos) [inline content/v2/domains/templates.markdown]: @jacegu I believe this is a common description in all pages. If we make any change, it should be in all pages.

Also, it should look like a welcoming description and end with a dot. It is likely the description that search engines will display.

I don't think it's reasonable to make any change in this PR.

File: layouts/v2-sidebar.html [hunk 3]
```diff
@@ -17,6 +17,7 @@ <h4 class="sidebar-title">API v2</h4>
   <li><ul class="nav-list">
     <li><a href="/v2/domains/certificates/">Certificates</a></li>
     <li><a href="/v2/domains/email-forwards/">Email Forwards</a></li>
+    <li><a href="/v2/domains/templates/">Templates</a></li>
```
Comment (jacegu) [inline layouts/v2-sidebar.html]: [There is](https://github.com/dnsimple/dnsimple-developer/blob/master/layouts/v2-sidebar.html#L37) already a Template link in the sidebar.

File: content/v2/domains/templates.markdown (hunk 2 above)
Comment (jacegu) [inline content/v2/domains/templates.markdown]: > I believe this is a common description in all pages. If we make any change, it should be in all pages.

It sucks but I'll 😶 then.

File: layouts/v2-sidebar.html (hunk 3 above)
Comment (jodosha) [inline layouts/v2-sidebar.html]: @jacegu You're right. I moved them under Templates section. The structure now reflects Services > Domains.

**Pull request merged**